src/main/java/school/lemon/changerequest/java/generics/container/GenericContainerDemo.java

```diff
@@ -0,0 +1,72 @@
+package school.lemon.changerequest.java.generics.container;
+
+public class GenericContainerDemo {
+    public static void main(String[] args) {
+        intDemonstration();
+        stringDemonstration();
+    }
+
+    public static void intDemonstration() {
+        System.out.println("Usage of int container");
+        GenericContainer<Integer> intContainer = new GenericContainer<Integer>();
+        System.out.println("Empty container:\n" + "Size: " + intContainer.size() + "\n" + intContainer);
```

Comment:
Try to avoid such concatenation constructions.

It'll be easier to read, if you'll use `String.format` instead.